### PR TITLE
Netia Ingram

### PR DESCRIPTION
**Pull request by netia1128**

### CODE REVIEW

File: lib/carnival.rb [hunk 1]
```diff
@@ -0,0 +1,83 @@
+require_relative 'attendee'
+require_relative 'ride'
+
+class Carnival
+    attr_reader :name,
```
Comment (mikedao) [inline lib/carnival.rb:5]: Indentation should be two spaces, not four.

File: lib/carnival.rb [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+
+    def add_ride(ride)
+        @rides << ride
+    end
+
+    def recommend_rides(attendee)
+        @rides.find_all do |ride|
+            attendee.interests.include?(ride.name)
+        end
+    end
```
Comment (mikedao) [inline lib/carnival.rb:27]: Great

File: lib/carnival.rb [hunk 3]
```diff
@@ -0,0 +1,83 @@
...
+        attendees.find_all do |attendee|
+            attendee.interests.include?(ride.name)
+        end
+    end
+
+    def ticket_lottery_contestants(ride)
+        attendees_interested_in_ride(ride)
+    end
+
+    def draw_lottery_winner(ride)
```
Comment (mikedao) [inline lib/carnival.rb:72]: This should be returning a string, and not an object.

File: lib/carnival.rb [hunk 4]
```diff
@@ -0,0 +1,83 @@
...
+        ticket_lottery_contestants(ride).sample
+    end
+
+    def announce_lottery_winner(ride)
+        if draw_lottery_winner(ride).nil?
+            "No winners for this lottery"
+        else
+            "#{draw_lottery_winner(ride)} has won the #{ride.name} ticket."
```
Comment (mikedao) [inline lib/carnival.rb:80]: This does not match the spec.

**Review by mikedao — COMMENTED** [review]
Overall, very solid work. Just make sure that you're following exactly what the spec is asking for, we have two mismatches here.

Score: 2.75

File: lib/carnival.rb (hunk 1 above)
Comment (netia1128) [reply lib/carnival.rb:5]: Mike - is this 2 space best-format across the board (my attr_readers, defs, etc)?

**Pull request closed without merging**